app/components/landscape-portrait-slider.js

```diff
@@ -1,32 +1,242 @@
     const classes = useStyles()
     const [isLandscape, setIsLandscape] = useState(true)
-
-    const handelInput = e => {
-        setIsLandscape(!isLandscape)
+    const { className, style, linkObj, portraitStatement } = props
+    const handelInputDesktop = () => {
+        setIsLandscape(true)
+    }
+    const handelInputPortrait = () => {
+        setIsLandscape(false)
     }
     return (
         <>
-            <iframe
-                src='https://cc.enciv.org/country:us/organization:ucla-student-accociation/office:usac-president/2021-05-07'
-                className={cx(isLandscape && classes.landscape, !isLandscape && classes.portrait)}
-            ></iframe>
+            <div className={classes.body}>
+                <div className={classes.portraitTailoredContainer}>
+                    <div
+                        className={cx(
+                            isLandscape && classes.landscapeContainer,
+                            !isLandscape && classes.portraitContainer
+                        )}
+                    >
+                        <iframe
+                            src='https://cc.enciv.org/country:us/organization:ucla-student-accociation/office:usac-president/2021-05-07'
+                            className={cx(isLandscape && classes.landscape, !isLandscape && classes.portrait)}
+                        ></iframe>
+                    </div>
+
+                    <div className={cx(isLandscape && classes.disabled, !isLandscape && classes.tailoredContainer)}>
+                        <h2 className={classes.tailoredHeader}>{portraitHeader}</h2>
+                        <p className={classes.tailoredParagraph}>{portraitParagraph}</p>
+                    </div>
+                </div>
+
+                <div className={classes.daContainer}>
+                    <SvgRectangle className={classes.iconda} />
+                    <p className={classes.iconda}> {daLink}</p>
+                    <p className={classes.icondaPerson}> Windspotter, Rabiechel</p>
+                </div>
+
+                <div className={classes.iconContainer}>
+                    <i
+                        onClick={handelInputDesktop}
+                        className={cx(!isLandscape && classes.noborderBottom, isLandscape && classes.borderBottom)}
+                    >
+                        <SvgDesktop className={classes.icon} />{' '}
+                    </i>
+                    <i
+                        onClick={handelInputPortrait}
+                        className={cx(isLandscape && classes.noborderBottom, !isLandscape && classes.borderBottom)}
+                    >
+                        <SvgPortrait className={classes.icon} />
+                    </i>
+                </div>
+            </div>
         </>
     )
 }
 
 //possibly set height to a ratio based on width (w/1920*1080)
 const useStyles = createUseStyles(theme => ({
+    body: {
+        backgroundColor: 'white',
```

Comment:
shouldn't set the backgroundColor let it come from the parent
I don't think max-width is doing anything here - take it out unless there's a reason.
while we are at it - lets not use body as a class name - body is an HTML tag.  It's not wrong, just confusing.